src/backend/gog/library.ts

```diff
@@ -316,7 +316,8 @@ export class GOGLibrary {
    */
   public async getInstallInfo(
     appName: string,
-    installPlatform = 'windows'
+    installPlatform = 'windows',
```

Comment:
I believe we could also use the platform we have on context, this could improve the re-render process I guess.